src/cargo/ops/cargo_install.rs

```diff
@@ -768,13 +781,36 @@ pub fn uninstall_one(
     config: &Config,
 ) -> CargoResult<()> {
     let crate_metadata = metadata(config, root)?;
-    let mut metadata = read_crate_list(&crate_metadata)?;
+    let metadata = read_crate_list(&crate_metadata)?;
+    let pkgid = PackageIdSpec::query_str(spec, metadata.v1.keys())?.clone();
+    uninstall_one_impl(crate_metadata, metadata, &pkgid, bins, config)
+}
+
+fn uninstall_cwd(
+    root: &Filesystem,
+    bins: &[String],
+    config: &Config,
+) -> CargoResult<()> {
+    let crate_metadata = metadata(config, root)?;
+    let metadata = read_crate_list(&crate_metadata)?;
+    let source_id = SourceId::for_path(config.cwd())?;
+    let (pkg, _source) = select_path_pkg(None, &source_id, None, config, true)?;
```

Comment:
This includes an error message that mentions `--path` and `--git` which don't exist in `uninstall`.